sdk/formrecognizer/ai-form-recognizer/CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # Release History
 
-## 3.0.0 (Unreleased)
+## 3.0.0-preview.1 (2020-08-11)
 
-- This release marks the general availability of the Azure Form Recognizer SDK for JavaScript.
+- Changed the package version to 3.0.0-preview.1 to reduce confusion with older versions of the Azure Form Recognizer SDKs.
```

Comment:
Should we say it is targeting v2.0?

Reply:
I added this to the changelog message (L7) in a previous commit.

Reply:
actually looks like it says that on line 7

Reply:
Should we make it the first line ?